easybuild/easyconfigs/x/XOOPIC/XOOPIC-20210302-foss-2020a.eb

```diff
@@ -0,0 +1,55 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'XOOPIC'
+version = '20210302'
+local_commit = '3fa6670'
+
+homepage = 'https://github.com/rinku-mishra/xoopic'
+description = """XOOPIC is a two-dimensional 3-velocity particle-in-cell simulator. 
+It can handle electrostatic and electromagnetic models, has a large variety of 
+boundary conditions, supports multiple gasses and gas chemistry, and is easily 
+reconfigurable via an input file."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/rinku-mishra/xoopic/archive/%s' % local_commit]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['e81a2b7a527b905fd58afd080f45690963079b0a0493a55929d2a0ef1246d4e3']
+
+builddependencies = [
+    ('binutils', '2.34'),
```

Comment:
no need to explicitly specify `binutils` when using the `foss` toolchain
```suggestion
```

Reply:
Fixed, thanks for pointing that out.